VoiceController.php

```diff
@@ -1,48 +1,50 @@
-public function voice(Request $request){
+public function voice(Request $request) {
     $request->validate([
-        'question_id'=>'required|int|exists:questions,id',
-        'value'=>'required|boolean',
+        'question_id' => 'required|int|exists:questions',
+        'value' => 'required|boolean',
     ]);
 
-    $question=Question::find($request->post('question_id'));
-    if (!$question)
+    try {
```

Comment:
@PiotrekPKP interesting attempt. But are you sure that your option is cleaner or shorter code with try-catch instead of regular if-check?

Reply:
It was the only thing I was concerned about acctually, maybe I'm wrong :)

Reply:
php artisan make:resource UserResource